modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts

```diff
@@ -176,6 +182,10 @@ const toggleFullscreen = (editor: Editor, fullscreenState: Cell<ScrollInfo | nul
     editorContainerStyle.top = fullscreenInfo.containerTop;
     editorContainerStyle.left = fullscreenInfo.containerLeft;
 
+    sinkContainerS.each((elm) => {
+      elm.dom.style.position = 'relative';
```

Comment:
```suggestion
      Css.set(element, 'position', 'relative');
```

Reply:
I wonder if it would be more safe if this would restore the state to what ever it was before we changed it rather than hard coding it.

- [ ] Also this needs a test could be a simple did the sink get the right state or not if the fullscreen was toggled with the `ui_mode` being set. Or go all advanced on it and measure things on screen.

Reply:
A simple test was added. The previous sink css position value is preserved within the FullScreenInfo state.